lib/target.js

```diff
@@ -12,6 +12,11 @@ module.exports = class Target {
       return acc + xs.groupCount
     }, 0)
     this.output = this.params.map(xs => [xs.name, null])
+    this.isConcat = (
+      this.route.length === 1 &&
+      this.route[0] === '/' &&
+      this.method === '*'
+    )
```

Comment:
It turns out that `* / foo` is :sparkles:_magic_:sparkles: — it means "concatenate these routes to the router", not "nest these routes." Thus we have three cases at 🔴.